cluster-sharding/src/main/scala/org/apache/pekko/cluster/sharding/external/ExternalShardAllocationStrategy.scala

```diff
@@ -44,6 +45,21 @@ object ExternalShardAllocationStrategy {
 
   type ShardRegion = ActorRef
 
+  /**
+   * Scala API: Create an [[ExternalShardAllocationStrategy]]
+   */
+  def apply(systemProvider: ClassicActorSystemProvider, typeName: String)(
+      implicit timeout: Timeout = 5.seconds): ExternalShardAllocationStrategy =
+    new ExternalShardAllocationStrategy(systemProvider, typeName)(timeout)
+
+  /**
+   * Java API: Create an [[ExternalShardAllocationStrategy]]
+   */
+  @varargs
+  def create(systemProvider: ClassicActorSystemProvider, typeName: String, duration: java.time.Duration)
+      : ExternalShardAllocationStrategy =
+    this.apply(systemProvider, typeName)(Timeout.create(duration))
```

Comment:
1. remove `@varargs` ,which is not needed.
2. the `Timeout.create(duration)` can use `duration.asScala` I think.